http/src/main/java/io/micronaut/http/form/FormConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.form;
+
+/**
+ * Configuration related to forms.
+ * @author Sergio del Amo
+ * @since 4.7.1
+ */
+public interface FormConfiguration {
+    /**
+     *
+     * @return default maximum of decoded key value parameters
+     */
+    int getMaxDecodedKeyValueParameters();
```

Comment:
specify here in the javadoc what is the default if not specific

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11272/commits/243edad24a190e265f08575c7168a993e7e94f35